omnibus/files/private-chef-cookbooks/private-chef/attributes/default.rb

```diff
@@ -116,6 +116,45 @@
 default['private_chef']['rabbitmq']['consumer_id'] = 'hotsauce'
 default['private_chef']['rabbitmq']['env_path'] = "/opt/opscode/bin:/opt/opscode/embedded/bin:/usr/bin:/bin"
 
+default['private_chef']['rabbitmq']['max_length'] = 100000
```

Comment:
Given that this is only applied to the /analytics vhost, let's call it something like `analytics_max_length`.  Another thought here is that instead of using a separate flag for `enabled` we could just check to see if the length is > 0.